pynest/nest/lib/hl_api_connections.py

```diff
@@ -634,14 +638,14 @@ def Disconnect(pre, post, conn_spec, syn_spec):
     sps(post)
     sr('cvgidcollection')
 
-    if conn_spec is not None:
-        sps(conn_spec)
-        if is_string(conn_spec):
-            sr("cvlit")
+    if is_string(conn_spec):
+        conn_spec = {'rule': conn_spec}
+    if is_string(syn_spec):
+        syn_spec = {'model': syn_spec}
+    if not syn_spec:
+        syn_spec = {'model': 'static_synapse'}
```

Comment:
@sanjayankur31 It is there in case somebody give an empty dictionary as `syn_spec`. I added this because of the examples given in #786, but when I think about it again now, it should not be possible to call `nest.Disconnect(nodes, nodes, 'all_to_all, {})`. If you don't want to specify `syn_spec` you should just call `nest.Disconnec(nodes, nodes, 'all_to_all'). I'll remove it now!